api/resources_portal/test/end_to_end/test_multiple_resources_requested_and_fulfilled.py

```diff
@@ -73,11 +73,41 @@ def test_create_account_and_list_resource(self, mock_auth_request, mock_record_r
         # Select two resources from PrimaryLab and request them, uploading two signed IRBs
         self.client.force_authenticate(user=requester)
 
+        # Upload IRBs
+        irb1 = Attachment(
+            filename="irb_attachment",
+            description="Institutional Board Review for the research in question.",
+            s3_bucket="a bucket",
+            s3_key="a key",
+        )
```

Comment:
I'm surprised this works if `owned_by_org` isn't set.

Reply:
It did not! It has since been updated.